Add docs for customizng agent-as-tool (#504)
openai/openai-agents-python: docs/tools.md

    print(result.final_output)
```

### Customizing tool-agents

> The `agent.as_tool` function is a convenience method to make it easy to turn an agent into a tool. It doesn't support all configuration though; for example, you can't set `max_turns`. For advanced use cases, it's very easy to configure it yourself:

Reviewer: ```suggestion
The `agent.as_tool` function is a convenience method to make it easy to turn an agent into a tool. It doesn't support all configuration though; for example, you can't set `max_turns`. For advanced use cases, use `Runner.run` directly in your tool implementation:
```
